pages/index.tsx

```diff
@@ -118,6 +118,21 @@ const useStyles = makeStyles(theme => ({
     padding: theme.spacing(10, 15, 3),
   },
 }))
+
+export interface TestRun {
+  test_run_id: number,
+  launch_id: number,
+  project_id: number,
+  project_name: string,
+  launch: string,
+  test_type: string,
+  start_datetime: string,
+  end_datetime: string,
+  duration: string,
```

Comment:
what happened with duration here? Why string?

Reply:
didn't understand a change in this interface, apart from `data: { url?: string }` to be changed to `test_run_data: {},`

Why does it show like fully new piece of code? I think `launch_name` should stay

Reply:
@opishcheiko mmmmmm will take a look into this.... thanks :)

Reply:
Good catch @opishcheiko it was just `launch` in the Core, so I changed it to `launch_name`, also updated the interface for `duration` and the optional `spectre_test_run_url`